test/CMakeLists.txt

```diff
@@ -59,7 +66,7 @@ if (BUILD_TESTS)
         ProcessorCount(N)
         set(CTEST_PARALLEL_LEVEL ${N} CACHE STRING "CTest parallel level")
         set(CTEST_TIMEOUT 90 CACHE STRING "CTest timeout")
-        add_custom_target(check ${CMAKE_CTEST_COMMAND} --output-on-failure -j ${CTEST_PARALLEL_LEVEL} -C ${CMAKE_CFG_INTDIR} --timeout ${CTEST_TIMEOUT}
+        add_custom_target(check-ctest ${CMAKE_CTEST_COMMAND} --output-on-failure -j ${CTEST_PARALLEL_LEVEL} -C ${CMAKE_CFG_INTDIR} --timeout ${CTEST_TIMEOUT}
```

Comment:
I would prefer to have `check` run ctest as currently does. I dont see the purpose of renaming this target.

Reply:
This is to align it with the `Makefile` for now. This target (among others) will be deprecated since it is totally unnecessary since you should just use the `ctest` command.

Reply:
Most developers(including me) use `make check` to run the tests with ctest. I really dont want to see it change to something that is suboptimal. 

Also, its shorter to do `make -j $(nproc) check` than doing `make -j $(nproc) cppcheck testrunner && ctest --output-on-failure -j $(nproc) .`. So I this shouldn't be removed either.